package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Authenticate to Google Cloud using OIDC tokens or JSON service account keys.",
   "main": "dist/main/index.js",
   "scripts": {
-    "build": "ncc build src/main.ts -o dist/main && ncc build src/post.ts -o dist/post",
+    "build": "ncc build -m src/main.ts -o dist/main && ncc -m build src/post.ts -o dist/post",
```

Comment:
The new version supports minifying the js, which makes the download much much smaller